packages/codegen-ui/lib/generate-form-definition/helpers/defaults.ts

```diff
@@ -34,6 +34,12 @@ export const FORM_DEFINITION_DEFAULTS = {
       value: 'fieldName',
       name: 'fieldName',
       valueMappings: { values: [{ value: { value: 'Option' } }] },
+      fileUploaderConfig: {
```

Comment:
Im guessing we couldnt rename fileUploader here to storage because of Model changes?

Reply:
yea we can't rename this one without changing model